saleor/graphql/order/tests/test_fulfillment.py

```diff
@@ -1811,13 +1812,16 @@ def test_fulfillment_approve_delete_products_before_approval_allow_stock_exceede
     response = staff_api_client.post_graphql(
         query, variables, permissions=[permission_manage_orders]
     )
-    content = get_graphql_content(response, ignore_errors=True)
-    errors = content["errors"]
-    assert len(errors) == 1
-    assert (
-        errors[0]["message"]
-        == "Insufficient stock for Test product (SKU_AA), Test product 2 (SKU_B)"
-    )
+
+    content = get_graphql_content(response)
+    errors = content["data"]["orderFulfillmentApprove"]["errors"]
+
+    assert len(errors) == 2
+    assert errors[0]["code"] == "INSUFFICIENT_STOCK"
```

Comment:
Are you sure that these errors are returned always in the same order? If we get the same result but in different order, this test will fail.

Reply:
I kept the order from old tests, but you are right, there is no `ordering` in `FulfillmentLine` model so I updated tests to ignore errors order.